Classes/Domain/Repository/DocumentRepository.php

```diff
@@ -215,177 +205,151 @@ public function findAllByCollectionsLimited($collections, $limit = 50)
     }
 
     /**
-     * Find all the titles
-     *
-     * documents with partof == 0
-     *
-     * @param array $settings
-     *
-     * @return array|\TYPO3\CMS\Extbase\Persistence\QueryResultInterface
-     */
-    public function findAllTitles($settings = [])
-    {
-        $query = $this->createQuery();
-
-        $constraints = [];
-        $constraints[] = $query->equals('partof', 0);
-
-        if ($settings['collections']) {
-            $constraints[] = $query->in('collections.uid', GeneralUtility::intExplode(',', $settings['collections']));
-        }
-
-        if (count($constraints)) {
-            $query->matching(
-                $query->logicalAnd($constraints)
-            );
-        }
-
-        return $query->execute();
-    }
-
-    /**
-     * Count the titles
-     *
-     * documents with partof == 0
+     * Count the titles and volumes for statistics
      *
-     * @param array $settings
-     *
-     * @return int
-     */
-    public function countAllTitles($settings = [])
-    {
-        return $this->findAllTitles($settings)->count();
-    }
+     * Volumes are documents that are both
+     *  a) "leaf" elements i.e. partof != 0
+     *  b) "root" elements that are not referenced by other documents ("root" elements that have no descendants)
 
-    /**
-     * Count the volumes
-     *
-     * documents with partof != 0
-     *
      * @param array $settings
      *
      * @return int
```

Comment:
Everything fine, except applying Scrutinizer fixes.

Reply:
Right. Thank you.